invenio_ldapclient/config.py

```diff
@@ -5,7 +5,19 @@
 # Invenio-LDAPClient is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
-"""Invenio v3 LDAP client for authentication and user attributes population."""
+"""Invenio v3 LDAP client for authentication and user attributes population.
+
+In your instance's ``config.py`` or via other means, you MUST override the
+following configuration:
+
+.. code-block:: python
```

Comment:
Yes it is needed, because documentation is auto generated from the code files. This tells the documentation renderer to render this part of the documentation with python syntax highlighting.

If you run `./run-tests.sh`, the documentation is built (`sphinx-build -qnNW docs docs/_build/html` in it is what does it). You can then open up the `docs/index.html` file in your browser to see it.